common/SmokeTests/SmokeTest/SmokeTest.csproj

```diff
@@ -10,13 +10,13 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Core" Version="1.8.0" />
+    <PackageReference Include="Azure.Core" Version="1.9.0" />
     <PackageReference Include="Azure.Identity" Version="1.4.0-beta.1" />
     <!-- The OverrideDailyVersion attribute prevents the Update-Dependencies script from overwriting it with a daily build version -->
```

Comment:
Does this mean that we can remove this override options now?

Reply:
Seems like it. `dotnet restore` works locally with `5.3.0` now.

Reply:
The GA release let us move the source back to a package reference from a project reference, since the processor no longer depends on unpublished changes in the main library.    I suspect that's the reason for the difference.